projects/manager/src/app/datasets/dataset-editor/dataset-editor.component.html

```diff
@@ -31,6 +31,25 @@
                         <mat-label>Dataset Description</mat-label>
                         <textarea matInput formControlName="description"></textarea>
                     </mat-form-field>
+                    <mat-form-field>
```

Comment:
When I add something, click apply, get the success message and go back, I have to tag.

![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/6e526ebd-9bc6-41fb-9bb0-98a36d0c227d)

Reply:
you had to press enter to actually add it to the list. I added the behavior on blur: 56872931bedfe4aeeccebf061c6a20fadb852bdf